chitchat_server/models.py

```diff
@@ -0,0 +1,53 @@
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import ForeignKey
+from sqlalchemy.orm import DeclarativeBase
+from datetime import datetime
+
+class Base(DeclarativeBase):
+    pass
+
+db = SQLAlchemy(model_class=Base)
+
+from sqlalchemy.orm import Mapped, mapped_column
+
+
+# These classes need to match the database tables specified in sql/scripts/init.sql in order
+# to work properly.
+
+# Additionally in the main Flask app, the following code needs to be run in order for the database to connect
+# correctly.
+# 
+# ```python3
+# app.config["SQLALCHEMY_DATABASE_URI"] = "postgresql://user:password@hostname"
+# db.init_app(app)
+# with app.app_context():
+#     db.reflect()
+# ```
+#
+# After these steps are followed the database can be interacted with via the below model classes and the 
+# db.session methods. See https://flask-sqlalchemy.readthedocs.io/en/stable/quickstart/ for examples.
+
+
```

Comment:
I will import these classes into the main server functionality so that it can interact with the database